src/main/java/com/fasterxml/jackson/core/StreamReadConstraints.java

```diff
@@ -13,17 +13,24 @@
     private static final long serialVersionUID = 1L;
 
     /**
-     * Default setting for maximum length: see {@link Builder#maxNumberLength(int)} for details.
+     * Default setting for maximum number length: see {@link Builder#maxNumberLength(int)} for details.
      */
     public static final int DEFAULT_MAX_NUM_LEN = 1000;
 
+    /**
+     * Default setting for maximum string length: see {@link Builder#maxStringLength(int)} for details.
+     */
+    public static final int DEFAULT_MAX_STRING_LEN = 1000000;
```

Comment:
One million (and 2 megs of memory) sounds like a reasonable starting point.

Could you just change this to use `1_000_000` for slightly easier reading? :)

Reply:
done